skynet/modules/stt/streaming_whisper/app.py

```diff
@@ -24,7 +25,7 @@ async def websocket_endpoint(websocket: WebSocket, meeting_id: str, auth_token:
                 log.info(f'Received disconnect message for {meeting_id}')
                 ws_connection_manager.disconnect(meeting_id)
                 break
-            await ws_connection_manager.process(meeting_id, chunk)
```

Comment:
Why don't we generate a UUID v7 right here right now? (rather in the process function I guess). Since it's the processing time. We use the time to then generate the ID... We could generate the ID and extract the time if we need to. But even generating the ID and then getting a new now() timestamp should be fine for our purpose.